src/components/SignoffSummary/index.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { Fragment } from 'react';
+import { makeStyles } from '@material-ui/styles';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import ListSubheader from '@material-ui/core/ListSubheader';
+import Typography from '@material-ui/core/Typography';
+import { object } from 'prop-types';
+
+const useStyles = makeStyles(theme => ({
+  listSubheader: {
+    lineHeight: 1.5,
+    marginBottom: theme.spacing(1),
+  },
+  listWrapper: {
+    display: 'flex',
+  },
+  requiresSignoffsList: {
+    marginRight: theme.spacing(6),
+  },
+}));
+
+function SignoffSummary(props) {
+  const classes = useStyles();
+  const { requiredSignoffs, signoffs } = props;
+  const listOfSignoffs = Object.entries(signoffs);
+
+  return (
+    <div className={classes.listWrapper}>
+      <List
+        dense
+        subheader={
+          <ListSubheader className={classes.listSubheader}>
+            Requires Signoffs From
+          </ListSubheader>
+        }>
+        {Object.entries(requiredSignoffs).map(([role, count], index) => {
+          const key = `${role}-${index}`;
+
+          return (
+            <ListItem key={key} dense className={classes.requiresSignoffsList}>
```

Comment:
Remove the `dense` prop from both `ListItem` components. Instead add `paddingTop: 0, paddingBottom: 0` to the `requiresSignoffsList` class.

Reply:
This caused a bit of inconsistency in the horizontally alignment. I ended wrapping the Required Signoffs text in a `<Typography>` to fix it.